### ISSUE TITLE
Improve UX on arrows when setting a time in the time picke on todos/meetings etc.

### ISSUE THREAD

**Issue by maxneuman** [Lundalogik/lime-elements#1545]
## New feature motivation

 
Users experience it to be unclear what happens when you click up/down :(

See image:
![image](https://user-images.githubusercontent.com/84496681/152343066-609d3ffb-da6f-4e0e-a9c3-d3bad1c3d5c3.png)

**Comment by adrianschmidt** [comment]
This seems like a problem that appears because the calendar view has been put next to the picker… When the picker is viewed in isolation, I can't imagine many users would find it logical that pressing _up_ would _decrease_ the value in the input… 🤔

I wholly agree that the date-picker is a component that could use a _lot_ of improvement, but I don't think a quick change to the functionality of the arrows will do much to help 😞 (Also, the arrows are a native feature of numeric input fields, so it's actually the browser adding those buttons and controlling what they do, not us.)

@Kiarokh and @LucyChyzhova, do you have any input?

**Comment by LucyChyzhova** [comment]
Yes, it seems, like the problem exists because we have the calendar view on aside. I have tested it a bit yesterday night and this morning and, I think, we could remove arrows from the time picker at all and leave the possibility to change time by selecting it from the keyboard or mouse scroll (please, take a look at the video:  https://web.microsoftstream.com/video/1edb16d8-2af8-46eb-a326-f5a871e29361) So the preview timeline will not move while you choosing the appropriate time it will change the place only by pressing 'enter' or selecting another element. 

Another problem that came to my mind is that the 'clear button' works for opening a date-picker and for a 'clear value' 👇https://web.microsoftstream.com/video/ea8890bb-a78e-4c92-950e-dbbaf4a994fd I think we need to fix it, and it should be that difficult

**Comment by Kiarokh** [comment]
Lucy :/ It says the video could not be found. 

🤔  I think this's the best we could do 👇 
![image](https://user-images.githubusercontent.com/35954987/152496480-827fca8c-1205-499b-a786-6f7dab93a5fe.png)

**Comment by maxneuman** [comment]
A bit tricky with the calendar on the side indeed but I think the +/- would be an improvement :)

**Comment by Kiarokh** [comment]
I just hope it's possible to do so. I think we can add some `before` and `after` pseudo elements with `+` and `-` as `content` and align them within the `input type="number"`. Also give them a background so they cover the original ones, and add `pointer-events: none;` so they can let through clicks on the original buttons below. 

We need to investigate if this works

**Comment by adrianschmidt** [comment]
> Also give them a background so they cover the original ones, and add `pointer-events: none;` so they can let through clicks on the original buttons below.

Another problem with that is that the buttons (or controls, rather, since we don't know what the browser might decide to render, if anything) can look different in different browsers and on different platforms.
